PhysicsTools/PatAlgos/plugins/PATIsolatedTrackProducer.cc

```diff
@@ -374,6 +391,21 @@ void pat::PATIsolatedTrackProducer::produce(edm::Event& iEvent, const edm::Event
 
     int trackQuality = gentk.qualityMask();
 
+    // get combined probQ and probXY (if they are available)
+    float probQonTrack = 0, probXYonTrack = 0, probQonTrackNoLayer1 = 0, probXYonTrackNoLayer1 = 0;
+    if (useSiPixelTrackProbQXY_) {
+      if (!gt2siPixelTrackProbQXY_.isUninitialized() && gt2siPixelTrackProbQXY->contains(tkref.id())) {
```

Comment:
I tried this in the first place, I got something like that I try to dereference a non-existing product or something like that, so that's why I included the `useSiPixelTrackProbQXY_`, and then it worked well for the old AOD cases